### DISCUSSION TITLE
Providing API documentation

### DISCUSSION THREAD

**Discussion by xpwmaosldk** [immich-app/immich#1149] [General]
Hello, thank you for creating such a great service. 
I would like to use this service for my family and friends, but I have difficulty operating the immich server. 
Personally, it takes a lot of time to understand the server and it is also not easy to customize. 
If immich aims to support perfect self-hosting, I think it would be good to provide basic servers and also document the server's API so that server administrators can replace and configure them in a server environment they are comfortable with (for example, springboot). 
What do you think?

**Comment by alextran1502** [comment]
The server api can be found on the website https://immich.app/docs/api

I think the idea is decent though since we are not the one working on that so we have no plan of supporting it

**Reply by xpwmaosldk** [reply]
Okay, thanks!
